rows.go

```diff
@@ -178,7 +182,11 @@ func parseSingleValue(columnType string, val interface{}) (driver.Value, error)
 	case intType:
 		return int32(val.(float64)), nil
 	case longType:
-		return int64(val.(float64)), nil
+		// long values as passed as strings by system engine
+		if unpacked, ok := val.(float64); ok {
+			return int64(unpacked), nil
+		}
+		return strconv.ParseInt(val.(string), 10, 64)
```

Comment:
Could we add parameters here so it's obvious what those numbers mean, e.g. `strconv.ParseInt(val.(string), base=10, bitSize=64)`, if the language allows this.

Reply:
Go doesn't have keyword arguments. I will add comments